CHANGELOG.md

```diff
@@ -4,6 +4,50 @@
 
 ### Grafana Mimir
 
+* [CHANGE] Alertmanager: the following metrics are not exported for a given `user` when the metric value is zero: #9359
+  * `cortex_alertmanager_alerts_received_total`
+  * `cortex_alertmanager_alerts_invalid_total`
+  * `cortex_alertmanager_partial_state_merges_total`
+  * `cortex_alertmanager_partial_state_merges_failed_total`
+  * `cortex_alertmanager_state_replication_total`
+  * `cortex_alertmanager_state_replication_failed_total`
+  * `cortex_alertmanager_alerts`
+  * `cortex_alertmanager_silences`
+* [FEATURE] Querier: add experimental streaming PromQL engine, enabled with `-querier.query-engine=mimir`. #9367 #9368 #9398 #9399 #9403 #9417 #9418 #9419 #9420
```

Comment:
All PRs that are included in v2.14.0-rc.0 should be mentioned in that CHANGELOG, even if the feature is experimental or incomplete. If a feature is incomplete, just mention it.

Reply:
Do you mean to copy all PRs to keep them in both 2.14-rc.0 and unreleased? Right now, I've split the list of PRs into those that part of `2.14-rc.0` and those are new. That's how it was for this feature for 2.13 also

Reply:
Those that are new (after 2.14-rc.0) are in `unreleased` section. I can confirm that #9367 and #3968 are only on `main` and not on `r309` (basis for 2.14 release).